src/drivers/mct/shr/seq_drydep_mod.F90

```diff
@@ -35,10 +35,11 @@ module seq_drydep_mod
 
   ! !PRIVATE ARRAY SIZES
 
-  integer, private, parameter :: maxspc = 100              ! Maximum number of species
-  integer, public,  parameter :: n_species_table = 77      ! Number of species to work with
+  integer, private, parameter :: maxspc = 160              ! Maximum number of species
+  integer, public,  parameter :: n_species_table = 252     ! Number of species to work with
   integer, private, parameter :: NSeas = 5                 ! Number of seasons
-  integer, private, parameter :: NLUse = 11                ! Number of land-use types
+  integer, public,  parameter :: NLUse = 11                ! Number of land-use types
+  integer, public,  parameter :: NPatch = 80               ! Number of patch types
```

Comment:
We are concerned about NLUse and NPatch being constants. In this context does NLUse need to be coordinated with CTSM land-use types? Or are they separate from CTSM? It looks like having NPatch as a constant is for the maximum number of different vegetation patches that might be in CTSM. If it's a max number for efficiency it would be good to have this have the ability to change as a non-crop simulation will only have up to 17, and actually crop simulations only actively use about 24.